src/mapml.css

```
@@ -776,3 +776,45 @@ label.mapml-layer-item-toggle {
   ) {
   outline: 0!important;
 }
+
+
+/**
+* Feature Index
+ */
+.mapml-feature-index-box {
+  margin: -36px 0 0 -36px;
+  width: 72px;
+  height: 72px;
+  left: 50%;
+  top: 50%;
+  content: '';
+  display: block;
```

```suggestion
.mapml-feature-index-box:not([hidden]) {
  margin: -36px 0 0 -36px;
  width: 72px;
  height: 72px;
  left: 50%;
  top: 50%;
  content: '';
  display: block;
```

This is needed in conjunction to [a previous suggestion](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/636/files#r771205223) because the UA styles `display: none` for `hidden` is overriden by the `display: block`. 

We can remove this (`:not([hidden])`) once [`[hidden] { display: none!important }`](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/621/files#diff-56ad14054d43b64aa4b9c722e112e44b8769885fad4e9fc9311b63fea37cf643R495-R497) is applicable (after changes from that PR is pulled in or this PR is merged into master).